### DISCUSSION TITLE
(VS2022 17.1.3) Copilot Open / Trigger options greyed out

### DISCUSSION THREAD

**Discussion by BlackxSnow** [community/community#14757] [Copilot Conversations]
I managed to resolve the issue with a reinstall but will post this here for posterity:

Copilot was working yesterday, however today the aforementioned buttons are greyed out and the extension has refused to work until I reinstalled as above.

The log only has these two lines:
GitHub Copilot 1.12.0.0 (v1.12.0.0@0660f7705)
GitHub Copilot Agent v1.12.0

Win 10 Pro 19044.1586
VS2022 17.1.3 as per title
Programming in C++

**Comment by AndreaSerraDev** [comment]
I'm having a very similar issue. It was working wonderfully until a few days ago. Can't even logout to log back in. VS 2022
![image](https://user-images.githubusercontent.com/118254498/206668038-4cb0601c-3a78-43f1-a38b-4fc485d37aa6.png)

**Reply by Akash1134** [reply]
Hi @AndreaSerraDev  , we're sorry that you cannot use Copilot with VS Code. Similar issues have been reported in the community before, I'd recommend that you refer to the steps mentioned in this [discussion](https://github.com/community/community/discussions/14252) and see if that helps resolve the issue. Thanks!

**Reply by AndreaSerraDev** [reply]
No luck. I deleted the mentioned files (actually there was only the hosts file, the other one was missing) and even tried reinstalling the extension after doing that, but it didn't work. Same greyed out options

**Reply by AndreaSerraDev** [reply]
And, I'm using Visual Studio 2022, not VS Code.

**Comment by RandyCh** [comment]
I disable the 'enable globally' button and then restart.
and Open / Trigger options become selectable.

**Reply by cfcljmu** [reply]
🙄 "Turn it off and on again"!

**Reply by geskin** [reply]
For me even the enable globally button is grayed out :/

**Comment by ayim** [comment]
Issue looks kind of stale but if either of you (@AndreaSerraDev @gen3vra @BlackxSnow) are still having issues with Copilot in Visual Studio, please reach out at aayim@microsoft.com

--
Aaron
Product Manager for GitHub Copilot in Visual Studio
aayim@microsoft.com

**Reply by fred-flowrl** [reply]
Also have the same issue:
<img width="280" alt="image" src="https://github.com/community/community/assets/145227471/6d2c991a-6d73-4ebc-aa0a-53bb7565f652">

**Reply by ayim** [reply]
This looks like a VSCode issue, passing this onto @Akash1134.
